indolaw-parser/laws/uu-2014-26-mod-clean.txt

```diff
@@ -0,0 +1,207 @@
+c.
+Pihak pemohon bantuan menyediakan fasilitas lokal dan pelayanan administrasi yang tepat dan efektif; dan
+d.
+Pihak pemberi dan penerima bantuan wajib mengkoordinasikan bantuan di wilayahnya masing-masing.
+9.
+Pengecualian dan Fasilitas dalam Ketentuan Pemberian Bantuan
+Persetujuan ASEAN memberikan pengecualian berupa pembebasan pajak dan fasilitas untuk memasukkan personel, peralatan, dan bahan agar pemberian bantuan efektif dan efisien.
+10.
+Transit Personel, Peralatan, dan Bahan dalam Ketentuan Pemberian Bantuan
+Pihak lain yang wilayahnya menjadi tempat transit wajib memberikan fasilitas bagi personel, peralatan, dan bahan yang dibutuhkan atau digunakan dalam pemberian bantuan.
+11.
+Kerja Sama Teknis
+Para Pihak wajib melakukan kerja sama teknis yang difasilitasi ASEAN Coordinating Centre untuk meningkatkan kesiapsiagaan dan mengurangi risiko terhadap lingkungan hidup dan kesehatan manusia antara lain:
+a.
+mobilitas sumber daya;
+b.
+standardisasi format laporan;
+c.
+pertukaran informasi, tenaga ahli, teknologi, teknik, dan keterampilan;
+d.
+perencanaan pelatihan, pendidikan, dan kampanye peningkatan kesadaran;
+e.
+pengembangan teknik pembakaran terkendali;
+f.
+pertukaran pengalaman dan informasi di antara lembaga penegak hukum;
+g.
+pengembangan pasar untuk pemanfaatan bio massa;
+h.
+pengembangan program pelatihan bagi pemadam kebakaran;
+i.
+memperkuat dan meningkatkan kapasitas teknis.
+12.
+Penelitian Ilmiah
+Penelitian ilmiah wajib dilakukan baik secara bersama-sama maupun antarnegara ASEAN maupun sendiri-sendiri untuk:
+a.
+mempromosikan dan mendukung program penelitian ilmiah dampak terhadap kesehatan masyarakat jangka panjang;
+b.
+mengembangkan cara, metode, teknik dan peralatan untuk pengendalian kebakaran lahan dan/atau hutan termasuk pemadaman kebakaran.
+Peraturan perundang-undangan nasional yang berkaitan dengan Persetujuan ASEAN, antara lain:
+1.
+Undang-Undang Nomor 41 Tahun 1999 tentang Kehutanan (Lembaran Negara Republik Indonesia Tahun 1999 Nomor 167, Tambahan Lembaran Negara Republik Indonesia Nomor 3888);
+2.
+Undang-Undang Nomor 18 Tahun 2004 tentang Perkebunan (Lembaran Negara Republik Indonesia Tahun 2004 Nomor 85, Tambahan Lembaran Negara Republik Indonesia Nomor 4411);
+3.
+Undang-Undang Nomor 32 Tahun 2004 tentang Pemerintahan Daerah (Lembaran Negara Republik Indonesia Tahun 2004 Nomor 125, Tambahan Lembaran Negara Republik Indonesia Nomor 4437);
+4.
+Undang-Undang Nomor 31 Tahun 2009 tentang Meteorologi, Klimatologi, dan Geofisika (Lembaran Negara Republik Indonesia Tahun 2009 Nomor 139, Tambahan Lembaran Negara Republik Indonesia Nomor 5058);
+5.
+Undang-Undang Nomor 32 Tahun 2009 tentang Perlindungan dan Pengelolaan Lingkungan Hidup (Lembaran Negara Republik Indonesia Tahun 2009 Nomor 140, Tambahan Lembaran Negara Republik Indonesia Nomor 5059).
+II.
+PASAL DEMI PASAL
```

Comment:
@pmlsslw This should be one line Mel hehehe (check out the original PDF or  is_start_of_penjelasan_pasal_demi_pasal)

Please change it:
a) in mod-clean.txt, mod.json in indolaw-parser/laws, and mod.json in indolaw-nextjs/laws
b) change this exact problem in the other UU you parsed too

(SIDE NOTE: actually we should hardcode the cleaning logic to never trigger on this line: right now the heuristics always asks whether to split this line for every UU even though the answer is always no)

Reply:
Okay!